freqtrade/exchange/exchange.py

```diff
@@ -772,6 +773,39 @@ def get_fee(self, symbol='ETH/BTC', type='', side='', amount=1,
             raise OperationalException(e) from e
 
 
+# Store Exchange objects created
+__exchanges: Dict[str, Exchange] = {}
+
+
+def get_exchange(config, exchange_name: Optional[str] = None) -> Exchange:
+    """
+    Return Exchange object for exchange with name `exchange_name`.
+    Creates Exchange if it was not created before.
+    """
+    from freqtrade.resolvers import ExchangeResolver
+
+    name = exchange_name or config['exchange']['name']
+    if name not in __exchanges:
+        logger.info(f"Creating Exchange object for '{name}'...")
+        __exchanges[name] = ExchangeResolver(name, config).exchange
+        logger.info(f"Exchange object for '{name}' created.")
+
+    return __exchanges[name]
+
+
+def delete_exchange(config, exchange_name: Optional[str] = None):
+    """
+    Delete Exchange object for exchange with name `exchange_name`.
+    """
+    name = exchange_name or config['exchange']['name']
+    logger.info(f"Deleting Exchange object for '{name}'...")
+    if name in __exchanges:
+        del __exchanges[name]
```

Comment:
this line is probably incorrect;
the whole `delete_exchange` function should probably be eliminated, since we do not really need to delete an Exchange object once it was created...